app/models/course.rb

```diff
@@ -4,4 +4,22 @@ class Course < ApplicationRecord
   has_many :enrollments
 
   delegate :title, to: :coding_class
+
+  def student_name_list
+    names_list = []
+    self.enrollments.each do |enrollment|
+      names_list << "#{enrollment.student.first_name} #{enrollment.student.last_name}"
+    end
+    names_list
```

Comment:
```suggestion

    names_list
```
nit: indentation before implicit return after a block